[Documentation] update From a URL page content (#3306)
bacalhau-project/bacalhau: docs/docs/setting-up/data-ingestion/from-url.md

To upload a file from a URL we will use the `bacalhau docker run` command.

```bash
bacalhau docker run \
    --id-only \
    --wait \
    --input https://raw.githubusercontent.com/filecoin-project/bacalhau/main/README.md \
    ghcr.io/bacalhau-project/examples/upload:v1
```

The job has been submitted and Bacalhau has printed out the related job id.

### Structure of the command

Let's look closely at the command above:

* `bacalhau docker run`: call to bacalhau using docker executor

> * `ghcr.io/bacalhau-project/examples/upload:v1`: the name and version of the docker image we are using

Reviewer: docker uses the word "tag" rather than "version" for the second part of the docker image. Should we not be consistent with that?

> * `--input https://raw.githubusercontent.com/filecoin-project/bacalhau/main/README.md \`: URL path of the input data volumes downloaded from a URL source. 

Reviewer: The \ here is just to escape the newline when typing multi-line commands in a terminal, we can probably remove that here.

Should this line also be swapped with the line above so the order of the explanation matches the order of the parameters in the example above?

> The `bacalhau docker run` command takes advantage of the `--input` parameter. This will download a file from a public URL and place it in the `/inputs` directory of the container (by default).Then we will use a helper container to move that data to the /outputs directory.

Reviewer: ```suggestion
The `bacalhau docker run` command takes advantage of the `--input` parameter. This will download a file from a public URL and place it in the `/inputs` directory of the container (by default). Then we will use a helper container to move that data to the /outputs directory.
```

:::tip
You can find out more about the [helper container in the examples repository](https://github.com/bacalhau-project/examples/tree/main/tools/upload) which is designed to simplify the data uploading process.
:::
:::info

> For more details, see the [CLI commands guide](../../dev/cli-reference/all-flags.md#docker-run)

Reviewer: Don't think this link is correct.

```suggestion
For more details, see the [CLI commands guide](../../cli-reference/all-flags.md#docker-run)
```

:::

## Checking the State of Your Jobs

- **Job status**: You can check the status of the job using `bacalhau list`, processing the json ouput with the `jq`:

```bash
bacalhau list $JOB_ID --output=json | jq '.[0].Status.JobState.Nodes[] | .Shards."0" | select(.RunOutput)'
```

When the job status is `Published` or `Completed`, that means the job is done, and we can get the results using the job ID.

- **Job information**: You can find out more information about your job by using `bacalhau describe`.

```bash
bacalhau describe  $JOB_ID 
```

- **Job download**: You can download your job results directly by using `bacalhau get`. Alternatively, you can choose to create a directory to store your results. In the command below, we removed a directory in case it was present before, created it and downloaded our job output to be stored in that directory.

```bash
rm -rf results && mkdir ./results
bacalhau get --output-dir ./results $JOB_ID 
```

## Viewing your Job Output

> Each job result contains `outputs` subfolder and `extiCode`, `stderr` and `stdout` files with relevant content. To view the execution logs execute following:

Reviewer: ```suggestion
Each job result contains an`outputs` subfolder and `exitCode`, `stderr` and `stdout` files with relevant content. To view the execution logs execute following:
```

```bash
head -n 15 ./results/stdout
```
And to view the job execution result (`README.md` file in the example case), which was saved as a job output, execute:
```bash
tail ./results/outputs/README.md
```

## Get the CID From the Completed Job

To get the output CID from a completed job, run the following command:

```bash
bacalhau list $JOB_ID --output=json | jq -r '.[0].Status.JobState.Nodes[] | .Shards."0".PublishedResults | select(.CID) | .CID'
```

> The job will upload the CID to the public storage via Estuary. We will store the CID in an environment variable so that we can reuse it later on.

Reviewer: Estuary has recently been deprecated.

```suggestion
The job will upload the CID to the public storage via IPFS. We will store the CID in an environment variable so that we can reuse it later on.
```
